packages/client/src/containers/RetroPage/RetroPage.jsx

```diff
@@ -97,6 +138,13 @@ function RetroPage() {
     }
   };
 
+  const renderTeamMembers = (members) =>
+    members.map((member) => (
+      <option key={member.id} value={member.id}>
+        {member.first_name} {member.last_name}
+      </option>
+    ));
```

Comment:
Any special reason for having this as a function instead of doing a `map` inside the `<select>` further down?

Reply:
Separated the mapping logic as its own function because it's easy to read and clear.